### DISCUSSION TITLE
Remove Jump to Top of Page when at the Top of the Page

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2254] [Ideas] [answered]
This is the littlest thing to fix but it bugs me so much so whatever reason. Their is a button that lets you jump to the top of the page however this button doesn't go away when you are at the top of the page. 
<img width="1552" alt="Screen Shot 2022-01-05 at 10 51 07 AM" src="https://user-images.githubusercontent.com/83455454/148256982-7e6c297a-6e4c-4194-afc9-ad8b3ac1cf32.png">
I know this is so dumb to fix and is a waste of time but it feels wrong for whatever reason to keep it there.

**Comment by Miodec** [comment, marked as answer]
Sure thing, i agree.
